src/meds_torch/configs/predict.yaml

```diff
@@ -9,6 +9,20 @@ defaults:
   - paths: default
   - extras: default
   - hydra: default
+  # experiment configs allow for version control of specific hyperparameters
+  # e.g. best hyperparameters for given model and datamodule
+  - experiment: null
+
+  # config for hyperparameter optimization
+  - hparams_search: null
```

Comment:
Why do you need this in a prediction config?

Reply:
Removed thanks!